MinimOsd_Extra/Panels.h

```diff
@@ -1286,9 +1286,12 @@ static void panGPSats(point p){
        gps_str = 0x20;
 
     if(has_sign(p))
-	OSD::write_S(gps_str);
+      osd.printf_P(PSTR("%2i%c"), osd_satellites_visible, gps_str);
+     else
+      osd.printf_P(PSTR("%2i"), osd_satellites_visible);
+	    //OSD::write_S(gps_str);
```

Comment:
osd.printf_P is a variadic function which is very expensive for the CPU, it should be avoided as possible. Why you try to remove cheap  OSD::write_S(gps_str) with .printf_P? You don't look for compiled size?